packages/n/nappgui/xmake.lua

```diff
@@ -0,0 +1,116 @@
+package("nappgui")
+    set_homepage("https://github.com/frang75/nappgui_src")
+    set_description("SDK for building cross-platform desktop apps in ANSI-C")
+
+    add_urls("https://github.com/frang75/nappgui_src.git")
+    add_urls("https://github.com/frang75/nappgui_src/archive/refs/tags/$(version).tar.gz")
+    add_versions("v1.4.1", "20919d6227d7c787f7ba6e96eb036db2a6d9f723372309030e17c808d7cad9aa")
+
+    add_configs("shared", { description = "Build shared library.", default = false, type = "boolean" })
+
+    add_deps("cmake")
+    if is_plat("windows") then
+        add_syslinks("user32", "ole32", "gdi32", "d2d1", "dwrite", "comctl32", "windowscodecs")
+    elseif is_plat("macosx") then
+        add_frameworks("CoreGraphics", "CoreText", "Foundation", "AppKit", "UniformTypeIdentifiers")
+    elseif is_plat("linux") then
+        add_deps("gtk+3", "glib")
+    end
+
+    on_install("linux", "macosx", "windows", function (package)
+        local configs = { "-DNAPPGUI_DEMO=NO" }
+        table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
+        table.insert(configs, "-DNAPPGUI_SHARED=" .. (package:config("shared") and "YES" or "NO"))
+        import("package.tools.cmake").install(package, configs)
+
+        --inc/osapp is needed or else compliation fails because it cant find osapp.h
+        package:add("includedirs", "inc", "inc/osapp")
+
+        --dylibs are built to `bin`
+        package:add("linkdirs", "bin")
+        package:add("rpathdirs", package:installdir("bin"))
+
+        --static libraries are built to `lib`
+        package:add("linkdirs", "lib")
```

Comment:
xmake will add it automatically, please remove it

Reply:
xmake didn't add it for the `bin` directory, for some reason the .dylibs are built into there